tool/tctl/common/auth_command_test.go

```diff
@@ -71,6 +71,14 @@ func TestAuthSignKubeconfig(t *testing.T) {
 		SigningAlg: types.CertAuthoritySpecV2_RSA_SHA2_512,
 	})
 	require.NoError(t, err)
+	user, err := types.NewUser("testuser")
```

Comment:
nit: vertical space to separate blocks?

```suggestion

	user, err := types.NewUser("testuser")
```